features/facebookImport/src/model/importers/language-and-locale-importer.js

```diff
@@ -71,7 +71,7 @@ export default class LanguageAndLocaleImporter {
             // TODO: Refactor how warnings are created within the importer.
             return {
                 status: IMPORT_WARNING,
-                importerClass: this.constructor.name,
+                importerClass: this.constructor,
```

Comment:
Is this what you want, [return a link to the object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/constructor)? Or the *name* of the object?

Reply:
Here we need the object at the moment. But all this will go away once we have a unified way to handle the status between analyses and importers.